### DISCUSSION TITLE
Identifying Items and Services for In-Network Rate File

### DISCUSSION THREAD

**Discussion by tdfow** [CMSgov/price-transparency-guide#28] [Q&A] [answered]
Within the mandate for the In-Network Rate File, it refers to reporting all items and services, does that mean that every item and service that is defined using the CMS code sets as the basis (based on billing code).   Is there any consideration for a Provider's license/specialty?   Or will all items and services from these code sets need to be included in the in-network rate file?
(https://www.cms.gov/files/document/code-sets.pdf)
Diagnoses (ICD-10-CM)
Procedures (ICD-10-PCS)
ICD-10 – International Classifcation of Diseases, 10th edition
Outpatient services/procedures CPT – Current Procedure Terminology
Health care equipment and supplies
Health Care Common Procedure Coding System (HCPCS)
Dental procedures CDT – Code on Dental Procedures and Nomenclature
Drug products NDC – National Drug Codes

**Comment by shaselton-usds** [comment, marked as answer]
You can read the definition here: https://www.federalregister.gov/d/2020-24591/p-1201

**Comment by ghost** [comment]
The list of allowed billing code types does not include "CDT", so dental would be out of scope?

**Reply by monty3301** [reply]
Most dental plans are not covered by the rule as most are excepted benefits.  But I agree there could be some dental covered.

**Comment by ryan-thomas-peterson** [comment]
@shaselton , @shaselton-usds , the definition of "all encounters, procedures, medical tests, supplies, prescription drugs, durable medical equipment, and fees (including facility fees), provided or assessed in connection with the provision of health care."  

The goal of this conversation and https://github.com/CMSgov/price-transparency-guide/discussions/64  are assessing what items can be 'provided or assessed' for an individual provider.  Certain providers can not under their licensure provide certain services.  Often times contracts don't designate what specific codes a provider can bill for due to maintenance considerations (an ever evolving list of codes).
